### ISSUE TITLE
1024x1024

### ISSUE THREAD

**Issue by xiangw369** [zhudongwork/SAM_TensorRT#2]
Does this algorithm process 1024x1024 ，no matter what size picture it processes ，Is the output image also that large?

**Comment by zhudongwork** [comment]
Yes, the input size of the model is 1024*1024. The images were processed using ResizeLongestSide-1024 in the original SAM project. In order to avoid tensorrt's dynamic inference of orig_img_size, the input size is fixed at 1024*1024. You can resize the output mask result to the original size of the image.
